Tests/SkillFunctionalTests/Utils/ConfigureAppSettings.ps1

```diff
@@ -0,0 +1,185 @@
+#>
+
+$PSVersion = $PSVersionTable.PSVersion;
+if ($PSVersion.Major -lt 7) {
+  Write-Host "`n==============================================================================================================================";
+  Write-Host "  This tool only supports " -ForegroundColor White -NoNewline;
+  Write-Host "PowerShell 7+" -ForegroundColor Magenta -NoNewline;
+  Write-Host "." -ForegroundColor White;
+  Write-Host "  PowerShell '" -ForegroundColor White -NoNewline;
+  Write-Host "$($PSVersion.Major).$($PSVersion.Minor)" -ForegroundColor Yellow -NoNewline;
+  Write-Host "' was detected!. To run this tool successfully, please upgrade PowerShell to version '" -ForegroundColor White -NoNewline;
+  Write-Host "7.0" -ForegroundColor Green -NoNewline;
+  Write-Host "' or higher." -ForegroundColor White;
+  Write-Host "  For more information, refer to the following documentation:" -ForegroundColor White;
+  Write-Host "   - https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-7.1" -ForegroundColor Cyan;
+  Write-Host "==============================================================================================================================`n";
+  exit 1;
+}
+
+$Inputs = @{};
+$RootFolder = Resolve-Path -Path "$PSScriptRoot\..";
+$Settings = @{
+  ResourceGroup      = 'BFFN'
+  GroupsSuffix       = @(
+    'DotNet'
+    'JS'
+    'Python'
+  )
+  AppSettingsPath    = "$RootFolder\appsettings.json";
+  AppSettingsDevPath = "$RootFolder\appsettings.Development.json";
+  BotSettings        = [System.Collections.Concurrent.ConcurrentDictionary[string, object]]::new()
+  BotResources       = [System.Collections.Concurrent.ConcurrentDictionary[string, object]]::new()
+}
+
+# Check if user is logged in
+$LoginOutput = az ad signed-in-user show --only-show-errors --output none 2>&1;
+if ($LoginOutput) {
+  $LoginOutput;
+  exit 1;
+}
+
+# Resource Group Input
+Write-Host "? " -ForegroundColor Green -NoNewline;
+Write-Host "Enter the Resource Group where the Bots are located: " -ForegroundColor White -NoNewline;
+Write-Host "default (" -ForegroundColor DarkGray -NoNewline;
+Write-Host $Settings.ResourceGroup -ForegroundColor Magenta -NoNewline;
+Write-Host ") " -ForegroundColor DarkGray -NoNewline;
+$Inputs.ResourceGroup = (Read-Host).Trim();
+
+# Resource Suffix Input
+do {
+  Write-Host "? " -ForegroundColor Green -NoNewline;
+  Write-Host "Enter the Suffix used for the Bot Resources: " -ForegroundColor White -NoNewline;
+  Write-Host "eg. (microsoft-396) " -ForegroundColor DarkGray -NoNewline;
+  $Inputs.ResourceSuffix = (Read-Host).Trim();
+  if ([string]::IsNullOrEmpty($Inputs.ResourceSuffix)) {
+    Write-Host "  - The Bot Resource Suffix must be provided" -ForegroundColor Red;
+  }
+  else {
+    break;
+  }
+} while ($true)
+
+if ([string]::IsNullOrEmpty($Inputs.ResourceGroup)) {
+  $Inputs.ResourceGroup = $Settings.ResourceGroup;
```

Comment:
Is this comparison case sensitive? should we care about user input being in any casing?

Reply:
We thought about this, and we decided to leave it as the user inputs the value, and let the check bot resources existence to evaluate this, if the user provides 'bffn' for example as RG, but the resource group it's called 'BFFN'. The --resource-group when getting the directline or the resource existence seems to be case-insensitive, therefore we let the commands take care of it